test/integration/Templates.test.ts

```diff
@@ -90,4 +92,25 @@ describe('Client - Templates', function () {
             expect(result.Message).to.eq('OK');
         });
     });
+
+    describe('push templates', () => {
+        it("non existing servers error", () => {
+            let pushRequest = new TemplatesPushRequest(0,1, false);
+
+            return accountClient.pushTemplates(pushRequest).then(()=> { }, error => {
+                expect(error.name).to.equal('ApiInputError');
+                expect(error.message).to.equal('The source and destination servers were not found.');
+            });
+        });
+
+        it('no templates with aliases', async () => {
+            const server = await client.getServer();
+            let pushRequest = new TemplatesPushRequest(server.ID,server.ID, false);
```

Comment:
Shouldn't the error be different when we push to the same server? Also - no space between arguments.

Reply:
Also `const` instead of `let` would be better.

Reply:
Fixed. Regarding error, it comes from API, will create a ticket for it to fine tune there